src/postgraphile-upsert.ts

```diff
@@ -318,8 +319,10 @@ function createUpsertField({
                 ].join(", ")}`
               );
             }
+            assert(table.namespace, "expected table namespace");
 
             const [constraintName] = matchingConstraint;
+            const ignoreUpdate = {};
```

Comment:
i suspect this is type `{}`, the literal type, and that `omit(...)` returns any? if so, can we type explicitly?

Reply:
i see what this is for. can we do something like:

```
const columnNamesSkippingUpdate = new Set<string>()
// snip snip

if (omit(...)) {
  columnNamesSkippingUpdate.add(attr.name)
}

// snip snip
.filter(col => columnNamesSkippingUpdate.has(col.names[0]))